features/url-rewriters/imgix.md

```diff
@@ -0,0 +1,37 @@
+# Imgix URL-rewriter
+
+ImageKit can translate transformations written in Imgix's syntax to ImageKit's syntax. Learn how to use a URL-rewriter on [this page](/features/url-rewriters).
+
+{% hint style="info" %}
+**Beta feature**\
+This feature is currently in beta. Some transformations may not work as expected.
+{% endhint %}
+
+## Supported Transformations
+
+The following Imgix transformations are supported for translation:
+
+| Imgix transform | Example translation | Remarks |
+| - | - | - |
+| <p>width</p> | <ul> <li> `w=100` => `w-100`</li> </ul> |  |
+| <p>height</p> | <ul> <li>`h=100` => `h-100`</li> </ul> |  |
+| <p>aspect ratio</p> | <ul> <li>`ar=2:3` => `ar-2-3`</li><li>`ar=3:1` => `ar=3-1`</li><li>`ar=2:4` => `ar=2-4`</li> </ul> | <ul>Aspect ratio will only work when we  pass either width or height (not both) and fit</ul> |
```

Comment:
Replace the note with "Aspect ratio only works in combination with fit and either height or width (not both)". In the current sentence, the phrase "when we  pass" is a bit fuzzy in meaning and colloquial. Also, there are two spaces between `when` and `we`. No room for even little errors like that in our public docs.

Reply:
done